docs/data-management/documentation.qmd

```diff
@@ -0,0 +1,116 @@
+
+Without documentation, projects lose their usefulness over time, as illustrated below [see also @Michener].
+
+![Image from [Michener et al. 1997](https://doi.org/10.1890/1051-0761(1997)007%5b0330:NMFTES%5d2.0.CO;2)](../../images/information_loss.png)
+
+## Types of documentation
+
+This summary can help guide team conversations around documentation strategies.
+
+### Preliminary
+
+__Preliminary__ documentation refers to early-stage descriptions created during the planning phase. This often includes a data management plan (DMP), which outlines data collection, organization, storage, and sharing strategies. 
+
+### Process 
+
+__Process__ documentation involves capturing step-by-step procedures and workflows used to collect, process, analyze, and model data, including:
+
+* Documenting raw data sources
+* Documenting methods with code comments
+* Documenting methods with version control tools
+
+### Product 
+
+__Product__ documentation provides information to accompany code and data of completed projects, ensuring usability and transparency. This documentation can be a part of or accompanying written products.
+
+## Core documentation 
+
+### The project-level README file {#project-readme}
+
+At a minimum, the project should have a README file in text or markdown format with the information listed below. This can be an evolving, living document. While it's technically __product documentation__, it's easiest to start it early in the project's development.
+
+* Project name and description
+* Project PI and contact information
+* List of staff responsible for data management and code development
+* Associated final product(s), date of release, and DOI (if applicable)
+* Link to published data/code (if applicable)
+* License associated with final product(s)
+* Nature of sensitive or proprietary data (if applicable)
+* Any other important notes for navigating folder or using data/code
+
+### Raw data
+
+__The source of all downloaded raw datasets should be documented in a README file.__
+
+::: {.callout-note}
+As described in the [Organization](../data-management/organization.qmd) section, __all raw data should be retained in its raw form and not directly modified__.
+:::
+
+Best practices:
+
+* The README file associated with a raw data file(s) should be located in the same folder as the raw data file.
+* Create README files for logical “clusters” of related files / data.
+* Name the README so that it is easily associated with the data file(s) it describes (e.g., `README_PRISM_Daily_Temperature.txt`).
+* Format README files consistently.
+* Write the README document in an plain text and open source file format, such as .txt or .md.
+
+Below is a README template and example. Include in the README file the information shown in the example.
+
+Filename: `README_PRISM_Daily_Temperature.txt`
```

Comment:
I think the convention is to have the filename to be `README.md` or `README.txt` and then to have all of the identifying information inside the file. The assumption is that the file should go everywhere the dataset does, so it's superfluous to re-identify it. Also, some tools (GitHub for example) treat files called `README.foo` specially.